app-backend/db/src/test/scala/com/azavea/rf/database/PropTestHelpers.scala

```diff
@@ -305,34 +322,43 @@ trait PropTestHelpers {
 
   def fixupTaskFeaturesCollection(
       tfc: Task.TaskFeatureCollectionCreate,
-      project: Project
+      project: Project,
+      statusO: Option[TaskStatus] = None
```

Comment:
I think renaming these variables to either `taskStatusOption` or just `taskStatus` would be good - it's more readable.